route/src/route_generator_worker.cpp

```diff
@@ -418,6 +424,20 @@ namespace route {
     
     bool RouteGeneratorWorker::spin_callback()
     {
+        if(localFunction()==true)
+        {
+           this->rs_worker_.on_route_event(RouteStateWorker::RouteEvent::ROUTE_INVALIDATION);
+           publish_route_event(cav_msgs::RouteEvent::ROUTE_INVALIDATION);
+           
+           auto route=routing(destination_points_in_map_.front(),
+                                std::vector<lanelet::BasicPoint2d>(destination_points_in_map_.begin() + 1, destination_points_in_map_.end() - 1),
+                                destination_points_in_map_.back(),
+                                world_model_->getMap(), world_model_->getMapRoutingGraph());
+
```

Comment:
There needs to be a check here for if routing succeeded or not. If it failed we need to log something and call the route_aborted callback. If it succeeds the state needs to transition from routing to following.

Reply:
Done.